### PR TITLE
BUG in regularized inverse 2D

### PR DESCRIPTION
**Pull request by edouardoyallon**
Addresses #542 and correct a bug to use this on a laptop.

### CODE REVIEW

File: examples/2d/regularized_inverse_scattering_MNIST_torch.py [hunk 1]
```diff
@@ -84,7 +86,7 @@ def forward(self, input_tensor):
 
     transforms_to_apply = transforms.Compose([
         transforms.ToTensor(),
-        transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))  # Pixel values should be in [-1,1]
+        transforms.Normalize((0.5,), (0.5,))  # Pixel values should be in [-1,1]
```
Comment (janden) [inline examples/2d/regularized_inverse_scattering_MNIST_torch.py]: Don't see how this comment makes any sense. It normalizes the input to have mean 0.5 and standard deviation 0.5, no?

**Comment by janden** [comment]
I should also say that this runs fine on the cluster here using the CPU.

File: examples/2d/regularized_inverse_scattering_MNIST_torch.py (hunk 1 above)
Comment (edouardoyallon) [reply examples/2d/regularized_inverse_scattering_MNIST_torch.py]: Is it better?

Comment (edouardoyallon) [reply examples/2d/regularized_inverse_scattering_MNIST_torch.py]: (I've simply removed this comment and explained it was simply a standard practice and for compatbility issues)

**Pull request merged**